src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.stream.Stream;
+
+public enum Rank {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
돈과 같이 1000단위로 구분은 다음과 같이 underbar(_)를 활용해 구현하면 코드 가독성이 좋아집니다.